dev/Common/TerminalVelocityFeatures-BadgeNotifications.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<features xmlns="http://microsoft.com/windowsappsdk/TerminalVelocity/20210729/TerminalVelocityFeatures.xsd">
+
+  <!-- Copyright (c) Microsoft Corporation. All rights reserved.
+
+     Licensed under the MIT License. See LICENSE in the project root for license information.
+
+-->
+
+  <!-- See docs/Coding-Guidelines/TerminalVelocity.md for more info.  -->
+
+  <!-- Enabled variants -->
+  <feature>
+    <name>Feature_BadgeNotifications</name>
```

Comment:
nit : Feature_LocalBadgeNotifications ?

Reply:
badge notification alone is considered local for push it will be completely different, and changes will be part of pushnotification directory rather than here